aws/resource_aws_instance.go

```diff
@@ -1356,7 +1356,7 @@ func InstanceStateRefreshFunc(conn *ec2.EC2, instanceID string, failStates []str
 	return func() (interface{}, string, error) {
 		instance, err := resourceAwsInstanceFindByID(conn, instanceID)
 		if err != nil {
-			if !isAWSErr(err, "InvalidInstanceID.NotFound", "") {
+			if !isAWSErr(err, ec2.UnsuccessfulInstanceCreditSpecificationErrorCodeInvalidInstanceIdNotFound, "") {
```

Comment:
I see in `ec2/api.go` `UnsuccessfulInstanceCreditSpecificationErrorCodeInvalidInstanceIdNotFound = "InvalidInstanceID.NotFound"` but even though the constant's value is what we want it's name suggests something to do with an error from getting an instance's credit specification.
I see nothing with leaving the original string here as we do for most other EC2 error codes.

Reply:
makes sense 👍 , will revert here and below